client/src/services/Authorized.js

```diff
@@ -0,0 +1,15 @@
+import { checkToken } from '@/services/auth.js'
+
+export const isAuthorized = function () {
+  return new Promise((resolve, reject) => {
+    checkToken()
+      .then((res) => {
+        if (res.data.id === window.localStorage.getItem('profileId')) {
```

Comment:
res.data.id === window.localStorage.getItem('profileId') if this statement equal to true then you could resolve this result